python/manageJobs.py

```diff
@@ -204,10 +209,22 @@ def manageJobs(argv=None):
     parser.add_option("-w", "--why", dest="why", default=False, action="store_true", help="show why a job was held (default = %default)")
     parser.add_option("-m", "--matched", dest="matched", default=False, action="store_true", help="show site and machine to which the job matched (default = %default)")
     parser.add_option("-p", "--progress", dest="progress", default=False, action="store_true", help="show job progress (time and nevents) (default = %default)")
+    parser.add_option("-X", "--xrootd-resubmit", dest="xrootdResubmit", default=False, action="store_true", help="resubmit the jobs based on where the input files are located (default = %default)")
+    group = OptionGroup(parser, "Site Specific Resubmit Options",
+                        "The options for resubmitting jobs based on where there input files are stored (-X, --xrootd-resubmit).")
+    group.add_option("-C", "--input-file-classad", dest="inputFileClassAd", default = "", type="string", help = "HTCondor ClassAd which contains the input file(s) being used within the job (default = %default)")
+    group.add_option("-D", "--dry-run", dest="dryRun", default=False, action="store_true", help="don't actually resubmit any jobs (default = %default)")
+    group.add_option("-K", "--log_key", dest="logKey", default = "", type="string", help="key to use to find the correct line(s) in the log file (default = %default)")

```

Comment:
I wanted the logs to possibly be a fallback, though I realize that's not how it's written right now. Would you prefer to have the fallback option, or would you like them to be exclusive? If we go the exclusivity route, things become hard because `--log_path` has a default path. Therefore I can't key on it. I'll probably just be keying on `--log_key` (no pun intended).

Reply:
If the logs are intended to be a fallback, that should be documented clearly.

Reply:
Changed so that the logs are not a fallback and are exclusive with `-C`.